Source/gg2/Objects/Weapons/Minigun.events/Begin Step.xml

```diff
@@ -18,7 +18,8 @@
 else if (ammoCount &lt;= maxAmmo and isRefilling)
     ammoCount += 1 * global.delta_factor;
 if(!readyToShoot and alarm[5] &lt; 25 and !isRefilling)
-    alarm[5] += 1;
+    if(global.run_virtual_ticks)
+        alarm[5] += 1;
```

Comment:
Looking into this more it doesn't seem to do what I initially thought lol so will just revert it.